packages/sdk/Dockerfile

```diff
@@ -132,20 +135,82 @@ EOF
 
 ################################################################################
 # postgresql initdb
-FROM postgres:${POSTGRES_VERSION} AS postgresql-initdb
+FROM ${POSTGRES_BASE_IMAGE} AS postgresql-initdb
 ENV POSTGRES_PASSWORD=password
 RUN /usr/local/bin/docker-ensure-initdb.sh postgres
 
+################################################################################
+# cartesi rollups-node target
+FROM rollups-node-base AS rollups-node
+ARG CARTESI_MACHINE_EMULATOR_VERSION
+ARG CARTESI_ROLLUPS_NODE_VERSION
+ARG TARGETARCH
+
+USER root
+ARG DEBIAN_FRONTEND=noninteractive
+RUN <<EOF
+apt-get install -y --no-install-recommends \
+    libslirp0 \
+    lua5.4 \
+    xz-utils
+rm -rf /var/lib/apt/lists/*
+EOF
+
+RUN <<EOF
+set -e
+addgroup --system --gid 102 cartesi
+adduser --system --uid 102 \
+    --disabled-login \
+    --gecos "cartesi user" \
+    --home /nonexistent \
+    --ingroup cartesi \
+    --no-create-home \
+    --shell /bin/false \
+    cartesi
+EOF
+
+RUN mkdir -p /tmp/.cartesi && chmod 1777 /tmp/.cartesi
+
+# Install cartesi-machine emulator
+RUN <<EOF
+curl -fsSL https://github.com/cartesi/machine-emulator/releases/download/v${CARTESI_MACHINE_EMULATOR_VERSION}/cartesi-machine-v${CARTESI_MACHINE_EMULATOR_VERSION}_${TARGETARCH}.deb \
+    -o /tmp/cartesi-machine.deb
+dpkg -i /tmp/cartesi-machine.deb
+rm /tmp/cartesi-machine.deb
+cartesi-machine --version-json
+EOF
+
+# Install cartesi-rollups-node
+RUN <<EOF
+curl -fsSL https://github.com/cartesi/rollups-node/releases/download/v${CARTESI_ROLLUPS_NODE_VERSION}/cartesi-rollups-node-v${CARTESI_ROLLUPS_NODE_VERSION%%-*}_${TARGETARCH}.deb \
+    -o /tmp/cartesi-rollups-node.deb
+dpkg -i /tmp/cartesi-rollups-node.deb
+rm /tmp/cartesi-rollups-node.deb
+mkdir -p /var/lib/cartesi-rollups-node/snapshots
+chmod 755 /var/lib/cartesi-rollups-node/snapshots
+chown cartesi:cartesi /var/lib/cartesi-rollups-node/snapshots
+cartesi-rollups-node --version
+EOF
+
+COPY --from=go-migrate /usr/local/bin/migrate /usr/local/bin/
+COPY --from=graphql /usr/local/bin/cartesi-rollups-graphql /usr/local/bin/
+COPY --from=graphql-migration /usr/share/cartesi/rollups-graphql/migrations /usr/share/cartesi/rollups-graphql/migrations
```

Comment:
I think migrations belongs to dev.
On production we'll probably prepare the database once separately.

Reply:
same related to `go-migrate`